cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -468,6 +462,55 @@ def _generate_rabbitmq_certs(self):
             sign_key=sign_key,
         )
 
+    def handle_certificates(self,
+                            installing,
+                            cert_src=None,
+                            key_src=None,
+                            ca_src=None):
+
+        cert_config = {
+            'cert_destination': constants.BROKER_CERT_LOCATION,
+            'key_destination': constants.BROKER_KEY_LOCATION,
+            'ca_destination': constants.BROKER_CA_LOCATION,
+            'owner': 'rabbitmq',
+            'group': 'rabbitmq',
+        }
+
+        if installing:
+            return self.use_supplied_certificates(**cert_config)
+        else:
+            src_certs = {'cert_src': cert_src,
+                         'key_src': key_src,
+                         'ca_src': ca_src}
+            cert_config.update(src_certs)
+            return self.configure_certs_in_their_locations(**cert_config)
+
+    def replace_certificates(self):
+        if certificates.needs_to_replace_certificates(
+                constants.NEW_BROKER_CERT_FILE_PATH,
+                constants.NEW_BROKER_CA_CERT_FILE_PATH):
+            cert_src, key_src, ca_src = self.validate_new_certs()
+            self.logger.info(
+                'Replacing certificates on the rabbitmq component')
+            self.handle_certificates(installing=False,
+                                     cert_src=cert_src,
+                                     key_src=key_src,
+                                     ca_src=ca_src)
+
+            service.reload(RABBITMQ, ignore_failure=True)
```

Comment:
are you sure reload is enough for rabbitmq?

Reply:
Changed to restart.